atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -236,10 +238,32 @@ def _make_labels(self):
 
         return labels
 
+    def get_fs_task_id(self):
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
+                fs_task_id = fs_result['filesystem-koji-task-id']
+            except KeyError:
+                self.log.error("%s: expected filesystem-koji-task-id in result",
+                               AddFilesystemPlugin.key)
+            else:
+                try:
+                    task_id = int(fs_task_id)
+                except ValueError:
+                    self.log.error("invalid taskID %r", fs_task_id, exc_info=1)
```

Comment:
This block (from line 245) doesn't get execution coverage during unit testing. I think we need to have additional tests in tests/test_orchestrate_build.py for that.